src/Veldrid.Tests/Veldrid.Tests.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.console" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="Xunit.SkippableFact" Version="1.3.6" />
```

Comment:
It's somewhat dependent on how the CI system handles it, but I find that marking tests as skippable on conditions like `Skip.IfNot(GD.Features.ComputeShader);` helps to differentiate e.g. 'yes, compute shaders work for this back end' vs. 'couldn't validate one way or another whether or not compute shaders work for this back end' at a glance.